config/routes.rb

```diff
@@ -3,4 +3,7 @@
 
   # Defines the root path route ("/")
   # root "articles#index"
+  get "/ingredients", to: "ingredients#index"
+  get "/recipes/:id", to: "recipes#show"
+  get "/recipes/:id", to: "recipe_ingredients#show"
```

Comment:
Overall these routes are restful, but line 8 is never going to be used because Rails will match the verb (`get`) and the path `"/recipes/:id"` from the top of the file down. 

Also, I'm not sure why this route was built, if we're not using the `recipe_ingredients` controller in any stories. I recommend only building a route when you absolutely need it.